packages/wonder-blocks-icon/src/types.ts

```diff
@@ -2,3 +2,8 @@
  * All the possible icon weights.
  */
 export type PhosphorIconAsset = PhosphorRegular | PhosphorBold | PhosphorFill;
+
+/**
+ * All the possible icon sizes.
+ */
+export type IconSize = "small" | "medium" | "large" | "xlarge";
```

Comment:
**question:** Since in TypeScript 5, all enums are union enums, I wonder if we should do an enum here instead of a string union? 🤷🏻 I don't feel strongly about it. Just asking the question.

Reply:
That's fair, I'll make the change. Thanks for the suggestion!

Reply:
After quickly making the change, I realized that this would mean that we'd have to replace a bunch of callers to make it compatible with this change. (same would happen with `Button`, `IconButton`, etc).